Jenkinsfile

```diff
@@ -42,7 +42,7 @@ pipeline {
             allowEmptyResults: false,
             testResults: '**/target/surefire-reports/*.xml, **/target/failsafe-reports/*.xml'
           )
-          publishCoverage adapters: [jacocoAdapter(mergeToOneReport: true, path: '**/target/site/**/jacoco.xml')]
+          recordCoverage(tools: [[parser: 'JACOCO', pattern: 'coverage/target/site/jacoco-aggregate/jacoco.xml', mergeToOneReport: true]], sourceCodeRetention: 'MODIFIED')
```

Comment:
I'm wondering about the pattern used? Shouldn't it be the same?
The build in https://ci.jenkins.io/job/Infra/job/plugin-health-scoring/job/PR-289/1/coverage/ seems to be using the version on the `main` branch.

Reply:
Oh right, don't mind my bad copy/paste from updating core 😅

Reply:
No problem. Thanks for the contribution